ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -191,12 +196,24 @@ private void setRouter(Properties props, MockServerLayout mockServerLayout, Noti
         cryptoJobHandler, accountService, mockTime, MockClusterMap.DEFAULT_PARTITION_CLASS);
   }
 
+  /**
+   * Setup test suite to perform a {@link Router#putBlob} call using the constant {@link #PUT_CONTENT_SIZE}
+   */
   private void setOperationParams() {
-    putBlobProperties = new BlobProperties(-1, "serviceId", "memberId", "contentType", false, TTL_SECS,
+    setOperationParams(PUT_CONTENT_SIZE, TTL_SECS);
+  }
+
+  /**
+   * Setup test suite to perform a {@link Router#putBlob} call.
+   * @param putContentSize the size of the content to put
+   * @param ttlSecs the TTL in seconds for the blob.
+   */
+  private void setOperationParams(int putContentSize, long ttlSecs) {
+    putBlobProperties = new BlobProperties(-1, "serviceId", "memberId", "contentType", false, ttlSecs,
         Utils.getRandomShort(TestUtils.RANDOM), Utils.getRandomShort(TestUtils.RANDOM), testEncryption);
     putUserMetadata = new byte[10];
```

Comment:
minor: if useMetadata length is changeable, we'd better replace `10` with a constant variable here